test_conformance/math_brute_force/reference_math.cpp

```diff
@@ -1641,6 +1641,7 @@ double reference_expm1(double x)
 double reference_fmax(double x, double y)
 {
     if (isnan(y)) return x;
+    if (isnan(x)) return y;
```

Comment:
For consistency, we should probably do the same in:
 - `reference_fmin`
 - `reference_fmaxl`
 - `reference_fminl`